docs/.sphinx/get_vale_conf.py

```diff
@@ -0,0 +1,151 @@
+#! /usr/bin/env python
+
+import os
+import shutil
+import subprocess
+import tempfile
+import sys
+import logging
+import argparse
+
+# Configure logging
+logging.basicConfig(
+    level=logging.INFO,
+    format='%(asctime)s - %(levelname)s - %(message)s',
+    datefmt='%Y-%m-%d %H:%M:%S'
+)
+
+SPHINX_DIR = os.path.join(os.getcwd(), ".sphinx")
+
+GITHUB_REPO = "canonical/praecepta"
```

Comment:
At some point Docs team should use "propero" software installation approach: deb/snap/pip/.... get_vale_conf.py is actually vale_install.py.

LGTM as unrelated, but worth polishing one day.